src/main/java/uk/gov/companieshouse/api/testdata/controller/TestDataController.java

```diff
@@ -73,4 +71,35 @@ public ResponseEntity<Void> delete(@PathVariable("companyNumber") String company
         return new ResponseEntity<>(HttpStatus.NO_CONTENT);
     }
 
+    @PostMapping("/user")
+    public ResponseEntity<UserData> createUser(@Valid @RequestBody(required = false) UserSpec request) throws DataException {
+        Optional<UserSpec> optionalRequest = Optional.ofNullable(request);
+        var spec = optionalRequest.orElse(new UserSpec());
+
+        var createdUser = testDataService.createUserData(spec);
+
+        Map<String, Object> data = new HashMap<>();
+        data.put("user email", createdUser.getEmail());
+        data.put("user id", createdUser.getUserId());
+        LOG.info("New user created", data);
+        return new ResponseEntity<>(createdUser, HttpStatus.CREATED);
+    }
+
+    @DeleteMapping("/user/{userId}")
+    public ResponseEntity<Map<String, Object>> deleteUser(@PathVariable("userId") String userId) throws DataException {
+        Map<String, Object> response = new HashMap<>();
+        response.put("user id", userId);
+        boolean deleteUser= testDataService.deleteUserData(userId);
+
+        if(deleteUser) {
+            LOG.info("User deleted", response);
+            return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+        }
+        else {
+            response.put("status", HttpStatus.NOT_FOUND.value());
```

Comment:
Would it make sense if we used the enum object? It would eventually call its toString method which would be something like "404 NOT FOUND"
```suggestion
            response.put("status", HttpStatus.NOT_FOUND);
```

Reply:
updated as per the suggestion